### ISSUE TITLE
NVDA reads HTML code when adding citations in word with Zotero

### ISSUE THREAD

**Issue by enessaribas** [nvaccess/nvda#8841]
### Steps to reproduce:

1. install zotero, and add at least 1 reference
2. type a random sentence in word, and then select add/edit citation.
  
### Actual behavior:
NVDA speaks the following HTML information, taken from log on debug mode.
code:
Speaking [u'data:application/xhtml+xml,%3C!DOCTYPE%20html%20PUBLIC%20%22-//W3C//DTD%20XHTML%201.0%20Strict//EN%22%20%22http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd%22%3E%3Chtml%20xmlns=%22http://www.w3.org/1999/xhtml%22%3E%3Chead%3E%3Clink%20rel=%22stylesheet%22%20type=%22text/css%22%20href=%22chrome://zotero/skin/integration.css%22/%3E%3Clink%20rel=%22stylesheet%22%20type=%22text/css%22%20href=%22chrome://zotero-platform/content/integration.css%22/%3E%3C/head%3E%3Cbody%20contenteditable=%22true%22%20spellcheck=%22false%22%20id=%22quick-format-editor%22/%3E%3C/html%3E  grouping  editable']

### Expected behavior:

 NVDA does not read this information, and anounces the element on this location of the screen.

### System configuration:
windows 10 1809, 64 bit

#### NVDA Installed/portable/running from source:
installed
#### NVDA version:
alpha-16164,8a7da1d4
#### Windows version:
windows 10
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no

**Comment by Adriani90** [comment]
@enessaribas are you still able to reproduce this with NVDA 2020.1?

**Comment by enessaribas** [comment]
hi,
I no longer have this software  installed. And did not use the classic  mode where this issue occured when I did have it. Someone else will have to test.

**Comment by Adriani90** [comment]
I am closing this as abandoned due to missing updates and given that no further requests from users have been submited in this regard. If someone is able to test and report an update, please feel free to comment and we can reopen.
